js_modules/dagster-ui/packages/ui-core/src/assets/useRecentAssetEvents.tsx

```diff
@@ -253,12 +254,31 @@ export const RECENT_ASSET_EVENTS_QUERY = gql`
     $after: String
     $cursor: String
   ) {
+    assetsLatestInfo(assetKeys: [$assetKey]) {
+      id
+      ...AssetLatestInfoFragment
+    }
+    assetNodeOrError(assetKey: $assetKey) {
+      ... on Error {
+        ...PythonErrorFragment
+      }
+      ... on AssetNode {
+        id
+        partitionStats {
+          numMaterialized
+          numMaterializing
+          numPartitions
+          numFailed
+        }
+      }
+    }
```

Comment:
I don't see where in this PR this part is used and it's a beefy add, can we rm?

Reply:
ah yeah I was planning to show this information somewhere but will omit it for now.